Docs/Usage.md

```diff
@@ -178,16 +178,26 @@ If you want to check in the `Package.resolved` file so that everyone is on the s
 
 > Note that Swift Packages don't work in projects with configurations other than `Debug` and `Release`. That limitation is tracked here bugs.swift.org/browse/SR-10927
 
-You can also include local Swift Packages by referencing them by paths in `localPackages`. When these have the same name as `packages` they will be used instead of the remote repos. This is useful for local development.
+You can also include local Swift Packages by referencing them by paths in `localPackages` and you can use them as `dependencies` same with `packages`. When these local packages in `localPackages` have the same name as remote repos in `packages`, they will be used instead of the remote repos. This is useful for local development.
 
 ```yml
+packages:
+  Yams:
+    url: https://github.com/jpsim/Yams
+    from: 2.0.0
 localPackages:
```

Comment:
What do you think of just having a single `packages` dictionary, and you can specify a `path` on a Package. A Package can then be an enum with either a `.remote(url: String, version: VersionRequirement)` or a `.local(path: String)`. If a definition had a path it would be local, otherwise would parse a remote package. The only downside is that you can't have a remote and a local at the same time for mirroring, but I'm not sure if that's important as you can just use a local package? Thoughts?

Reply:
I agree 👍👍👍 . It is great to make the format more consistent if it is okay to only use `packages`. I really want to do so. 

However, IMO it would be better to do it later. I also not sure of the importance of the existing local package feature. However, it would affect backward compatibility so I want to set as deprecated at first before deleting.

Reply:
Or I have already prepared an implementation on local so can I make another PR after this merged?

Reply:
Thank you! I'll make PR ASAP after it merged 👍